gap/FinSetsForCAP.gi

```diff
@@ -685,6 +685,30 @@ AddIsSplitMonomorphism( FinSets,
     return IsInitial( Range( phi ) ) or ( not IsInitial( Source( phi ) ) and IsMonomorphism( phi ) );
 end );
 
+##
+AddIsLiftable( FinSets,
+  function ( beta, alpha )
```

Comment:
Could you swap alpha and beta (here and also below)? The CAP documentation uses alpha as the first argument, and using a different order makes it hard to think about this code.

Reply:
This is the convention of my lecture. The lift is the solution of the system χ α = β.